profile/2020-09-01-hybrid.json

```diff
@@ -0,0 +1,625 @@
+        },
+        {
+          "resourceType": "locations/vmSizes",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "operations",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "virtualMachines",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "virtualMachines/extensions",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "virtualMachineScaleSets",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "virtualMachineScaleSets/extensions",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "virtualmachineScaleSets/networkInterfaces",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "virtualMachineScaleSets/virtualMachines",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        },
+        {
+          "resourceType": "virtualMachineScaleSets/virtualMachines/networkInterfaces",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json"
+        }
+      ],
+      "2019-03-01": [
+        {
+          "resourceType": "disks",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/disk.json"
+        },
+        {
+          "resourceType": "snapshots",
+          "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/disk.json"
+        }
+      ]
+    },
+    "microsoft.databoxedge":{
+      "2019-08-01": [
```

Comment:
CLI doesnt use the latest api version 2019-08-01

Reply:
WE dont need to update CLI for azurestack profile sake. We will use the CLI whatever the apiversion  used by CLI for azurestack as well